memory_test.go

```diff
@@ -498,104 +498,104 @@ func TestMemArraysWithLenThreshold(t *testing.T) {
-		},
-		{
-			desc: "mixed array under threshold",
-			script: `y = []
-			y.push([]);
-			let i = 0;
-			checkMem();
-			for(i=0;i<100;i++){
-				y[0].push(i<50?0:true);
-			};
-			checkMem()`,
-			threshold: 200,
-			memLimit:  memUsageLimit,
-			expectedSizeDiff: SizeEmptyStruct + // Array overhead
-				(50 * SizeNumber) + (50 * SizeBool) + // (450) size of property values
-				// stack difference in going from
-				//	[..other, []]
-				//  to
-				//	[..other, true, 50]
-				+1,
-		},
+		// {
+		// 	desc: "array of numbers under length threshold",
+		// 	script: `y = []
+		// 	y.push([]);
+		// 	let i=0;
+		// 	checkMem();
+		// 	for(i=0;i<20;i++){
+		// 		y[0].push(i);
+		// 	};
+		// 	checkMem()`,
+		// 	threshold: 100,
+		// 	memLimit:  memUsageLimit,
+		// 	expectedSizeDiff: SizeEmptyStruct + // Array overhead
+		// 		20*SizeNumber, // size of property values
+		// },
+		// {
+		// 	desc: "array of numbers over threshold",
+		// 	script: `y = []
+		// 	y.push([]);
+		// 	let i=0;
+		// 	checkMem();
+		// 	for(i=0;i<200;i++){
+		// 		y[0].push(i);
+		// 	};
+		// 	checkMem()`,
+		// 	threshold: 100,
+		// 	memLimit:  memUsageLimit,
+		// 	expectedSizeDiff: SizeEmptyStruct + // Array overhead
+		// 		200*SizeNumber, // size of property values
+		// },
+		// {
+		// 	desc: "mixed array under threshold",
+		// 	script: `y = []
+		// 	y.push([]);
+		// 	let i = 0;
+		// 	checkMem();
+		// 	for(i=0;i<100;i++){
+		// 		y[0].push(i<50?0:true);
+		// 	};
+		// 	checkMem()`,
+		// 	threshold: 200,
+		// 	memLimit:  memUsageLimit,
+		// 	expectedSizeDiff: SizeEmptyStruct + // Array overhead
+		// 		(50 * SizeNumber) + (50 * SizeBool) + // (450) size of property values
+		// 		// stack difference in going from
+		// 		//	[..other, []]
+		// 		//  to
+		// 		//	[..other, true, 50]
+		// 		+1,
+		// },
 		{
 			desc: "mixed array over threshold",
 			script: `y = []
 			y.push([]);
 			let i = 0;
 			checkMem();
-			for(i=0;i<100;i++){
-				y[0].push(i<50?0:true);
+			for(i=0;i<20;i++){
+				y[0].push(i<10?0:true);
 			};
 			checkMem()`,
-			threshold: 50,
+			threshold: 10,
 			memLimit:  memUsageLimit,
 			expectedSizeDiff: SizeEmptyStruct + // Array overhead
-				(50 * SizeNumber) + (50 * SizeBool) + // (450) size of property values
-				// stack difference in going from
-				//	[..other, []]
-				//  to
-				//	[..other, true, 50]
-				+1,
-		},
-		{
-			desc: "mixed scattered array over threshold wcs",
-			script: `y = []
-			y.push([]);
-			let i = 0;
-			checkMem();
-			for(i=0;i < 100;i++){
-				y[0].push(i%10==0?0:true);
-			};
-			checkMem()`,
-			threshold: 50,
-			memLimit:  memUsageLimit,
-			expectedSizeDiff: SizeEmptyStruct + // Array overhead,
-				(100 * SizeNumber) + // size of property values
+				(10 * SizeNumber) + (10 * SizeBool) + // (450) size of property values
 				// stack difference in going from
 				//	[..other, []]
 				//  to
 				//	[..other, true, 50]
 				+1,
 		},
+		// {
```

Comment:
[q] same as above, but this looks like a new test that maybe is no longer necessary.